plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/DynamicFilteringJdbcSessionProperties.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.plugin.jdbc;
+
+import com.google.common.collect.ImmutableList;
+import io.airlift.units.Duration;
+import io.trino.plugin.base.session.SessionPropertiesProvider;
+import io.trino.spi.connector.ConnectorSession;
+import io.trino.spi.session.PropertyMetadata;
+
+import javax.inject.Inject;
+
+import java.util.List;
+
+import static io.trino.plugin.base.session.PropertyMetadataUtil.durationProperty;
+
+public class DynamicFilteringJdbcSessionProperties
+        implements SessionPropertiesProvider
+{
+    private static final String DYNAMIC_FILTERING_WAIT_TIMEOUT = "dynamic_filtering_wait_timeout";
```

Comment:
there should be a feature toggle too, e.g: `dynamic_filtering_enabled`

Reply:
Hi @sopel39 , thanks for the comment, but as mentioned [here](https://trino.io/docs/current/admin/dynamic-filtering.html) 

>Dynamic filtering is enabled by default using the `enable-dynamic-filtering` configuration property. To disable dynamic filtering, set the configuration property to `false`. Alternatively, use the session property `enable_dynamic_filtering`.

I think it is already there?

Reply:
This is global parameter, we should also have specific one for connector

Reply:
Done, is `TestJdbcWithoutDynamicFiltering` needed?